cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -1,22 +1,27 @@
-
 {% extends 'layout-2-1-bleedbar.html' %}
 
 {% import 'molecules/related-posts.html' as related_posts with context %}
 
 {% if not page %}
-      {% set doc = get_document('posts', 'four-years-working-for-you') %}
```

Comment:
Mostly it makes reviewing harder because it's a big diff for little change, if any change.